api/proto/disperser/disperser.proto

```diff
@@ -78,12 +78,12 @@ message AuthenticationData {
 
 message DisperseBlobRequest {
 	// The data to be dispersed.
-	// The size of data must be <= 2MiB. Every 32 bytes of data chunk is interpreted as an integer in big endian format
+	// The size of data must be <= 16MiB. Every 32 bytes of data chunk is interpreted as an integer in big endian format
 	// where the lower address has more significant bits. The integer must stay in the valid range to be interpreted
 	// as a field element on the bn254 curve. The valid range is 
 	// 0 <= x < 21888242871839275222246405745257275088548364400416034343698204186575808495617
-	// containing slightly less than 254 bits and more than 253 bits. If any one of the 32 bytes chunk is outside the range, 
-	// the whole request is deemed as invalid, and rejected. 
+	// which is a 254 bit number (meaning the first 2 bits of each chunk must always be 00).
```

Comment:
This isn't accurate though, `21888242871839275222246405745257275088548364400416034343698204186575808495617` is smaller than `2^254-1` (i.e. the original comment is accurate that it doesn't have 254 bits).

Also shall we not use `chunk` here since it's easy to confuse with the encoding chunks (which will become more user visible in the future)?

Reply:
Took me a while to figure this one out haha. Doesn't matter that it's < 2^254-1, what matters is that it's > 2^253-1, and so actually needs 254 bits.

The number is 254 bits according to https://hackmd.io/@jpw/bn254 (which also explains why its called bn254 and not bn253). Here's also a python script showing that it's a 254 bit number:
```
number = 21888242871839275222246405745257275088548364400416034343698204186575808495617
binary_representation = bin(number)

# Count the number of bits (excluding the '0b' prefix)
bit_count = len(binary_representation) - 2

print(f"The number is: {number}")
print(f"Binary representation: {binary_representation}")
print(f"Number of bits: {bit_count}")
```

Reply:
What's a better word that you would recommend instead of `chunk`? @bxue-l2 maybe you have an idea since you initially wrote this comment I believe?

Reply:
EDIT: based on bowen's comment below, I think I understand what you guys are saying now. Still think my comment is more accurate and the original one weird/wrong (a number cannot have more than 253 bits and less than 254 bits, it has a fixed number of bits, and could happen to have 254 bits).
Updated the comment order to reflect you guys' confusion, hopefully this helps: [6b78c52](https://github.com/Layr-Labs/eigenda/pull/780/commits/6b78c526afb2f451149007cc80a4859b9269a504)

Reply:
> (meaning the first 2 bits of each chunk must always be 00)

Pedantically, this is not true. Disperser will happily encode & disperse a blob that does not prefix chunks with 00 as long as all 32 byte chunks are in range. The 00 prefix is just the easiest way to achieve lossless chunk encoding.

Reply:
That's not pedantic that's important info! So @bxue-l2 was the original comment wrong then? Disperser doesn't check range?

Reply:
@pschork maybe wording is getting everyone confused? By "chunk" in here we are talking about 32byte field elements. If those are not prefixed by 00, then they are surely a number > the max allowed FE, and so therefore wouldn't be in range, right?

Reply:
When encoding, if your chunk size is **32 bytes** and the current chunk is within range, you don't need to pad it, but if given chunk exceeds the range then you need to truncate the chunk and pad it with zero to fit within range. The problem is when do this, you won't be able to decode the encoding because you wont know which chunks were or were not padded. To solve this you set the chunk size to **31 bytes** and reserve the *1 byte* as a prefix with `\x00` so that, when decoding, you can identify the start of a chunk. That is why each chunk (typically) starts with 00.

Reply:
https://github.com/Layr-Labs/eigenda/blob/master/encoding/utils/codec/codec.go

Reply:
please avoid using chunk here, it'll quickly get overloaded and confusing.

I think we don't need to mention it's prefixed with 00 (it's not precise anyway), we just need to mention each 32 bytes should fall in that value range so it's a valid field element.

Reply:
[795868e](https://github.com/Layr-Labs/eigenda/pull/780/commits/795868e27c83f5701419f5e42c72237b64e4de26)